testing/src/test/scala/org/http4s/Http4sSpec.scala

```diff
@@ -72,6 +72,7 @@ trait Http4sSpec
       p: Parameters,
       f: FreqMap[Set[Any]] => Pretty): Fragments = {
     addFragment(ff.text(s"$name  ${props.name} must satisfy"))
+    addBreak
```

Comment:
Inserts a break between a particular test name and all its cases. Without this break output for the first test case continues on the same line.